ecosystem.config.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
```

Comment:
`pm2-runtime ecosystem.config.js --only app-server` to run just one app, or drop the `--only` flag to run everything. This config allows us to combine cluster mode logs (not used atm) and write to file so Datadog can ingest logs.